Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -199,12 +197,11 @@ namespace EMStudio
     void AnimViewportWidget::UpdateCameraFollowUp(bool followUp)
     {
         const auto lookAndOrbitCameras =
-            AZStd::vector<AZStd::shared_ptr<AzFramework::CameraInput>>{ m_lookRotateCamera, m_lookTranslateCamera,
+            AZStd::vector<AZStd::shared_ptr<AzFramework::CameraInput>>{ m_lookTranslateCamera,
```

Comment:
I'm not sue this is the right change, this will stop rotation working for the free look camera when you're flying around with WASD

Reply:
m_lookRotateCamera is still added to the camera controls on init, just that we don't remove it when the follow up option changed.

Reply:
This still needs to be here as the camera will be removed when we go to follow, and then added back when we leave follow - I'm 99% sure with this change if you go to follow, then go back to normal (uncheck follow) then RMB to free look will not work.